prov/usnic/src/usdf_ep_dgram.c

```diff
@@ -322,7 +322,7 @@ static struct fi_ops_msg usdf_dgram_ops = {
 	.sendv = usdf_dgram_sendv,
 	.sendmsg = usdf_dgram_sendmsg,
 	.inject = usdf_dgram_inject,
-	.senddata = usdf_dgram_senddata,
+	.senddata = fi_no_msg_senddata,
```

Comment:
I haven't looked -- can you mix dgram and msg func pointers like this?  (I don't know if the signatures are the same, offhand)

Same question for the RDM function pointers, below.